### ISSUE TITLE
Performance problems comparing to other chart libs

### ISSUE THREAD

**Issue by mburger81** [swimlane/ngx-charts#248]
**I'm submitting a ...**  (check one with "x")
```
It's neither a bug nor a feature request.
I tested your chart lib and I like them, but I have some performance problems.
```

**Current behavior**
In the attached screenshot you can see two line charts with the same data. The one at the bottom is using ng2-charts (canvas) and the second is using your ngx-charts (svg)

The line chart is a power meter chart with about 250 records. The data is set at the same moment for both cahrts. The ng2-chart is renderer immediately the ngx-charts is renderer about 2 seconds later so a lot!

The data passed to the charts is a live date, so anytime there is new data passed, the chart is rerendered (this could be done better but this should not point of discussion in this post). The ng2-chart is rerendered immediately, it feels not rerendered but only as it ads only one new data.
The ngx-charts on the contrary is rerendered very slowly and it flickers any time it is rendered again.

So in your opinion what do you think, could I do something to fast it up? Is canvas faster then svg? Could there a performance problem on the lib?

![image](https://cloud.githubusercontent.com/assets/3778892/23309377/bdf3422a-faae-11e6-9ce9-cac112a85dbf.png)

* **ngx-charts version:** 4.1.3

* **Angular version:** 2.4.8

* **Browser:** [Chrome 54]

**Comment by mburger81** [comment]
The performance problems is discussed on this issue
https://github.com/swimlane/ngx-charts/issues/249

**Comment by marjan-georgiev** [comment]
This has been addressed in version 6.0.0.

Here's a plunkr with 1000 data points: https://plnkr.co/edit/hTPaexbMyYbX8xJC38Ja?p=preview
